### PR TITLE
Feature flags page in B3

### PR DESCRIPTION
**Pull request by benrudolph**
@biyeun was bored on the :airplane: and converted this page to B3. not sure if it's totally necessary, but thought i'd throw it up.
screen shot 2015-10-19 at 12 48 38 am

cc: @TylerSheffels

### CODE REVIEW

**Comment by TylerSheffels** [comment]
Looks good to me :+1:

**Comment by czue** [comment]
lol. :+1: from me but will let @biyeun review

File: corehq/apps/domain/templates/domain/admin/feature_flags.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends "settings/base_template.html" %}
+{% extends "settings/base_template_b3.html" %}
```
Comment (biyeun) [inline corehq/apps/domain/templates/domain/admin/feature_flags.html]: what i've been doing for dual templates is `settings/base_template.html` ->  `settings/bootstrap2/base_template.html` and then the b3 templates in `settings/bootstrap3/base_template.html` that way it's clear that, hey, this page is obv still using b2 when there is probably a b3 template available.

**Comment by biyeun** [comment]
:heart: :heart_eyes: :heart:

File: corehq/apps/domain/templates/domain/admin/feature_flags.html (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/domain/templates/domain/admin/feature_flags.html]: right on, will update

**Comment by benrudolph** [comment]
updated, visited as many of those pages as i could, trusting find replace skills

**Comment by biyeun** [comment]
@benrudolph can you resolve conflicts pls? :heart_eyes_cat:

**Comment by benrudolph** [comment]
accidentally committed pillowtop ref.. but should be g2g now!

**Pull request merged**